pkg/odo/cli/add/binding/binding.go

```diff
@@ -19,7 +19,7 @@ import (
 const BindingRecommendedCommandName = "binding"
 
 var addBindingExample = ktemplates.Examples(`
-# Add binding between service named 'myservice' and the component present in the working directory in the interactive mode
+# Add binding between a service, and the component present in the working directory in the interactive mode
```

Comment:
IMO (and I am not an English expert), a `,` before "and" is used when there are a list of items. Here you are categorically talking about "service" and "component". Hence, no need of a `,` here. Happy to be corrected.
```suggestion
# Add binding between a service and the component present in the working directory in the interactive mode
```